data/templates/sangoma.tmpl

```diff
@@ -64,6 +64,9 @@
         <P8684 para="DhcpVlan">{{ vlan_dhcp_enable }}</P8684>
         <!--FunctionKeys-->
         {% for idx in 1..cap_linekey_count %}
+        {% if _context["linekey_type_#{idx}"] == "queuetoggle" %}
+            {{ _context["linekey_value_#{idx}"] = queuetoggle }}
+        {% endif %}
         {{- sangoma.pcode("LineKey#{idx}.Type", sangoma.linekey_type_map(_context["linekey_type_#{idx}"]), '0') }}
         {{ sangoma.pcode("LineKey#{idx}.Value", _context["linekey_value_#{idx}"], '') }}
```

Comment:
To simplify the IF above, we can define a local variable and assign the conditional value to it, instead of mangling a variable inherited from the DB.

```suggestion
        {{ sangoma.pcode("LineKey#{idx}.Value", linekey_value, '') }}
```

Reply:
I think in this case queuetoggle code (e.g. *45) will never be present into the value field.
This suggestion will put empty string in case linekey_value#{id} is not present, but we need to put the queuetoggle code into the field